docker/compose/docker-compose.yml

```
@@ -3,11 +3,13 @@ services:
     build:
       context: .
       dockerfile: docker/dockerfiles/Dockerfile.frontend
+      args:
+        - NEXT_PUBLIC_BACKEND_URL=${NEXT_PUBLIC_BACKEND_URL} # Pass the variable as a build arg
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)